CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the `IDYNTREE_USES_ASSIMP` option on Windows if `IDYNTREE_USES_YARP` is also enabled (https://github.com/robotology/idyntree/pull/832).
 - Fixed the ``OBJ`` meshes visualization in the Visualizer library (https://github.com/robotology/idyntree/pull/833).
 ## [3.0.0] - 2020-02-03
+- Fixed `ModelVisualization::getWorldLinkTransform()` method (https://github.com/robotology/idyntree/issues/836)
+- Fixed the `IDYNTREE_USES_IRRLICHT` option when `irrlicht` is installed via vcpkg (https://github.com/robotology/idyntree/pull/806).
```

Comment:
iDynTree 3.0.0 has been already released, so any new changelog item needs to go under the `Unreleased` section, that will be come the 3.0.1 release section once we release it. See https://keepachangelog.com/en/1.0.0/#effort for more details.

Reply:
Sorry, I missed the `## [3.0.0] - 2020-02-03` line in the changelog and I messed up stuff